apps/protocol-frontend/src/components/DaoSettingsLayout.tsx

```diff
@@ -9,6 +9,8 @@ import DaoNameUpdateForm from './DaoNameUpdateForm';
 import DaoSettingsMemberUpdateForm from './DaoSettingsMemberUpdateForm';
 import DaoSettingsMembersTable from './DaoSettingsMembersTable';
 import ErrorView from './ErrorView';
+import FeatureFlagWrapper from './FeatureFlagWrapper';
+import VerificationFramework from './VerificationFramework';
 
```

Comment:
(Arbitrary comment)

I think this was outside of the scopes of the ticket, but it should probably default to an empty state where framework is none and the number input is hidden

Reply:
We can chose to do this in this pr or to create a new ticket

Reply:
Great call. I'll work on that tomorrow. Let's do it as a separate PR since I'll prob hack on a few designs

Reply:
Going to make another ticket for this 👍